addons/website/static/src/js/content/inject_dom.js

```diff
@@ -24,6 +24,8 @@ document.addEventListener('DOMContentLoaded', () => {
         htmlEl.dataset.country = country;
     }
 
+    htmlEl.dataset.logged = !!session.user_id;
```

Comment:
? is_website_user ? boolean to say if logged or not...
But at this end, it will be the same...

Reply:
Yep indeed, I thought about it during my review too but could not see the difference (but indeed should use is_website_user for the sake of clarity).